hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/SteadyStateThrottlingTest.java

```diff
@@ -146,13 +141,7 @@ final Stream<DynamicTest> checkBalanceQps() {
     @HapiTest
     @Order(7)
     final Stream<DynamicTest> restoreDevLimits() {
```

Comment:
(Hmm, maybe they are "developer" limits not "default" limits after all)

Reply:
Yes, the `@HapiTest` target networks have higher dev throttle limits.